DFE-Digital/Dfe.Academies.External
# 104226 application declaration page

Build declaration page:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/104226?McasTsid=26110

And Declaration summary page:-
https://dfe-gov-uk.visualstudio.com.mcas.ms/Academies-and-Free-Schools-SIP/_workitems/edit/104225?McasTsid=26110

## Type of change

## Expected behaviour





## Steps to reproduce issue (if relevant)
n/a

## Steps to test this PR
1. Run unit tests to ensure they still work, including new one's. Done by CI / CD
2. Go into an application, click on declaration. Ensure Declaration summary page displays correctly as per ticket
3. Click on 'Start section' ensure declaration page displays correctly as per ticket

## Prerequisites
n/a
